scripts/installation/install-openkat.yml

```diff
@@ -0,0 +1,531 @@
+#
+# Copyright (C) 2023  Jeroen Baten <jbaten@i2rs.nl>
+#
+#
+# Licensed under the EUPL, Version 1.2 or – as soon they will be approved by the
+# European Commission - subsequent versions of the EUPL (the "Licence");
+#
+# You may not use this work except in compliance with the Licence.
+# You may obtain a copy of the Licence at:
+#
+# https://joinup.ec.europa.eu/software/page/eupl5
+#
+# Unless required by applicable law or agreed to in writing, software distributed under the Licence is distributed on an "AS IS" basis,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the Licence for the specific language governing permissions and limitations under the Licence.
+#
+# Changelog
+# Date       Author   Comments
+# ===========================================================================
+# 2023-08-12 J. Baten Initial version
+# 2023-08-14 J. Baten Debugging. Hacking Superadmin password to support
+#                     current stable standard Ansible
+# 2023-09-16 J. Baten Inject variables for version numbers, and prepare for MR.
+# 2023-11-19 J. Baten Handling MR feedback.
+# 2023-12-06 J. Baten Handling more MR feedback.
+# 2023-10-13 J. Baten Handling more MR feedback.
+
+# Usage: Make sure you can ssh to the root account on the remote server using
+#        authorized_keys. After that run the following command (and don't forget
+#        the comma!
+#
+#        ansible-playbook -vvv install-openkat.yml -i <hostname or ip>, -u root
+
+---
+
+- name: Playbook to install OpenKAT 1.10.1 on Debian 12
+  hosts: all
+  become: true
+  become_user: root
+
+  vars:
+    katrelease: "1.11.0"
```

Comment:
1.14 has been released since this Pr

Reply:
Of course it has. This PR is getting old. Trying to keep up with a complete OpenKAT team is like playing wackamole. Also, stuff changes between releases.
Kind request to accept this PR so I can start working on the update.